src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js

```diff
@@ -97,11 +97,11 @@ const ActiveEmployeesTable = ({
         checkFilterValues(filterInputs.officeLocation, empl.officeLocation) &&
         checkFilterValues(
           filterInputs.year,
-          empl.endDate ? empl.endDate[0] : null
+          empl.entryDate ? empl.entryDate[0] : null
```

Comment:
endDate into entryDate?

Reply:
Yes, it is ActiveEmployeesTable, so year/month should filter according to entry date (those employees don't even have end dates)